src/pages/products/Products.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import ProductList from './components/ProductList';
+import './Products.scss';
+
+const Products = () => {
+  const [productList, setProductList] = useState([]);
+  const [tabState, setTabState] = useState({
+    tabBest: true,
+    tabSummer: false,
+  });
+
+  const handleTabChange = () => {
+    setTabState(tabState);
+  };
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/products/products.json')
+      .then(res => res.json())
+      .then(productsData => setProductList(productsData));
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    fetch('/data/products/products.json')
      .then(res => res.json())
      .then(productsData => setProductList(productsData));
  }, []);
```
목데이터를 사용하는 경우 시작하는 "/"의 의미가 public을 가리키는 것이기 때문에
`localhost:3000`은 생략하셔도 똑같이 작동합니다!